Bottom: 69415a28efff0c2cde93af2e70b889bab4f010cd
Top:    40559755ac3ae93f13572f97d69f3dba95d61ffb
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-09-28 13:09:13 +0800

Refresh of e1000-use-alias-for-default

---

diff --git a/hw/alpha/dp264.c b/hw/alpha/dp264.c
index 27bdaa1..e993be6 100644
--- a/hw/alpha/dp264.c
+++ b/hw/alpha/dp264.c
@@ -87,7 +87,7 @@ static void clipper_init(MachineState *machine)
 
     /* Network setup.  e1000 is good enough, failing Tulip support.  */
     for (i = 0; i < nb_nics; i++) {
-        pci_nic_init_nofail(&nd_table[i], pci_bus, "e1000", NULL);
+        pci_nic_init_nofail(&nd_table[i], pci_bus, "e1000-82540em", NULL);
     }
 
     /* IDE disk setup.  */
diff --git a/hw/ppc/ppc440_bamboo.c b/hw/ppc/ppc440_bamboo.c
index ada676c..f45fc1c 100644
--- a/hw/ppc/ppc440_bamboo.c
+++ b/hw/ppc/ppc440_bamboo.c
@@ -246,7 +246,7 @@ static void bamboo_init(MachineState *machine)
         for (i = 0; i < nb_nics; i++) {
             /* There are no PCI NICs on the Bamboo board, but there are
              * PCI slots, so we can pick whatever default model we want. */
-            pci_nic_init_nofail(&nd_table[i], pcibus, "e1000", NULL);
+            pci_nic_init_nofail(&nd_table[i], pcibus, "e1000-82540em", NULL);
         }
     }
